testing/cross_browser.rake

```diff
@@ -0,0 +1,65 @@
+BROWSERSTACK_AUTHKEY = 'abcd1234'
+BROWSERS = JSON.parse(File.read(Rails.root.join('spec', 'browsers.json')))
+JAVASCRIPT_DRIVER = 'chrome_osx'
+
+desc 'Run all cross browser tests in parallel'
+task :cross_browser, [:files] => [:environment, 'cross_browser:tunnel', 'cross_browser:close_tunnel'] do
```

Comment:
Possibly, but we would need a way to let developers configure how to run their test suites. This is a simpler version of what we're using on CVP where I removed the Circle specific reporting bits. It would need to be tailored depending on your CI service.